src/code/fault_drawer.c

```diff
@@ -94,40 +94,40 @@ void FaultDrawer_DrawRecImpl(s32 xStart, s32 yStart, s32 xEnd, s32 yEnd, u16 col
     }
 }
 
-#ifdef NON_MATCHING
-// regalloc and minor ordering differences
 void FaultDrawer_DrawChar(char c) {
     u16* fb;
     s32 x, y;
+    s32 test;
```

Comment:
Did that temp var end up necessary? If so I think it deserves a proper name, or at least something like `temp`.